manager/manager.go

```diff
@@ -218,7 +221,7 @@ func New(config *Config) (*Manager, error) {
 	m := &Manager{
 		config:      config,
 		listeners:   listeners,
-		caserver:    ca.NewServer(RaftNode.MemoryStore(), config.SecurityConfig),
+		caserver:    ca.NewServer(RaftNode.MemoryStore(), config.SecurityConfig, caReconciliationRetryInterval),
```

Comment:
Well, I think testing this is very important, otherwise we risk breaking it without noticing. And without the param, a trivial unit test to cover it adds 10 seconds to `make test`. So I think it's worth having the parameter to allow test coverage without this overhead.